docs/Guide.RunningOnCI.md

```diff
@@ -80,3 +82,95 @@ script:
 - detox test --configuration ios.sim.release --cleanup
 
 ```
+
+### • Running Detox on [Bitrise](https://www.bitrise.io/)
+
+Bitrise is a popular CI service for automating React Native apps. If you are looking to get started with Bitrise, check out [this](http://blog.bitrise.io/2017/07/25/how-to-set-up-a-react-native-app-on-bitrise.html) guide.
+
+You can run Detox on Bitrise by adding a `tests` workflow. Here's what the Bitrise **.yml** file looks like for doing so:
+
+```yml
+---
+format_version: 1.1.0
+default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
+workflows:
+  _tests_setup:
+    steps:
+    - activate-ssh-key: {}
+    - git-clone:
+        inputs:
+        - clone_depth: ''
+    - script@1.1.4:
+        inputs:
+        - content: |-
+            #!/bin/bash
+            npm cache verify
+            npm install
+        title: Install Packages
+    before_run: 
+    after_run: 
+  _detox_tests:
+    before_run: []
+    after_run: []
+    steps:
+    - npm:
+        inputs:
+        - command: install -g detox-cli
+        title: Install Detox CLI
+    - npm@0.9.0:
+        inputs:
+        - command: install -g react-native-cli
+        title: Install React Native CLI
+    - script:
+        inputs:
+        - content: |-
+            #!/bin/bash
+            brew tap facebook/fb
+            export CODE_SIGNING_REQUIRED=NO
+            brew install fbsimctl
+            brew tap wix/brew
+            brew install applesimutils --HEAD
+        title: Install Detox Utils
+    - script@1.1.4:
+        title: Start Packager in Background
+        inputs:
+        - content: |-
+            #!/bin/bash
+            npm run start &
+    - script:
+        inputs:
+        - content: |-
+            #!/bin/bash
+            detox build --configuration ios.sim.release
+        title: Build Detox app
+    - script:
+        inputs:
+        - content: |-
+            #!/bin/bash
+            detox test --configuration ios.sim.release
+        - is_debug: 'yes'
```

Comment:
Should the debug option really be set here?

Reply:
What does `is_debug` mean ?

Reply:
not really @DanielMSchmidt 
@rotemmiz `is_debug` basically allows the step to `print debug infos about the working dir, tmp file path, exit code, etc`

I had that in there, when I was trying to debug it and get it to work. I can disable it now.